modules/cpon.js

```diff
@@ -114,6 +119,19 @@ CponReader.prototype.skipWhiteIsignificant = function()
 				this.ctx.getByte();
 				continue;
 			}
+			// skip BOM characters
+			else if(b === B) {
```

Comment:
I think that you can ignore all charactes with ASCII greater than 127
```js
		else if(b > 127) {
			this.ctx.getByte();
			continue;
		}
```

Reply:
Ok, even better. Changed and comitted.